### PR TITLE
Hide Gym Highlight

### LINKED ISSUE #114: Hide Gym Highlight

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #114 .

### CODE REVIEW

File: extension/scripts/features/hidegymhighlight/ttHideGymHighlight.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+body:not(.tt-mobile) .tt-available {
```
Comment (DeKleineKobini) [inline extension/scripts/features/hidegymhighlight/ttHideGymHighlight.css:1]: light theme should also have `:not(.dark-mode)`

File: extension/scripts/global/functions/torn.js [hunk 2]
```diff
@@ -260,3 +260,7 @@ function getPage() {
 function isCaptcha() {
 	return !!document.find(".captcha");
 }
+
+function darkMode() {
```
Comment (DeKleineKobini) [inline extension/scripts/global/functions/torn.js:264]: Should have a proper name, like `hasDarkMode`.

File: extension/scripts/features/hidegymhighlight/ttHideGymHighlight.css (hunk 1 above)
Comment (Sashank999) [reply extension/scripts/features/hidegymhighlight/ttHideGymHighlight.css]: Done.

File: extension/scripts/global/functions/torn.js (hunk 2 above)
Comment (Sashank999) [reply extension/scripts/global/functions/torn.js]: Changed.

File: extension/scripts/features/hidegymhighlight/ttHideGymHighlight.css (hunk 1 above)
Comment (Sashank999) [reply extension/scripts/features/hidegymhighlight/ttHideGymHighlight.css]: Added.

File: extension/scripts/features/hidegymhighlight/ttHideGymHighlight.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+			storage: ["settings.pages.sidebar.hideGymHighlight"],
+		},
+		async () => {
+			await requireSidebar();
+		}
+	);
+
+	function hideGymHighlight() {
+		const navGym = document.find("#nav-gym");
+		const gymClass = [...navGym.classList].find((name) => name.includes("available___"));
```
Comment (DeKleineKobini) [inline extension/scripts/features/hidegymhighlight/ttHideGymHighlight.js:21]: `navGym.classList.contains("^=available___")` is better 
It's a custom edit of the `contains` function.

**Comment by Sashank999** [comment]
All changes have been made. Tested on Chrome Mobile emulation.

**Review by DeKleineKobini — APPROVED** [review]

File: extension/scripts/features/hidegymhighlight/ttHideGymHighlight.js (hunk 3 above)
Comment (Sashank999) [reply extension/scripts/features/hidegymhighlight/ttHideGymHighlight.js]: Added.

**Pull request merged**